src/_help/enhance-documentation-content/topics.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Topics
+---
+
+- TOC
+{:toc}
+
+Topics are an exclusive custom property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
+
+By default, most specification standards do not support the addition of extra content. That's why we've added this additional property to insert at the beginning of your specification file: `x-topics`.
```

Comment:
```suggestion
By default, most specification standards do not support the addition of extra content, so we added this additional property to insert at the beginning of your specification file: `x-topics`.
```

Reply:
Not 100% sure of the transition here and that would make a super long sentence.

Reply:
That was just a suggestion because the rhythm of that part sounded a bit weird to me but you make the call. :)